install/installer/pkg/components/ide-metrics/configmap.go

```diff
@@ -200,6 +200,11 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 					AllowValues: []string{"*"},
 				},
 			},
+			Client: &config.ClientAllowList{
+				Name:         "metric_client",
+				AllowValues:  []string{"vscode", "supervisor", "unknown"},
```

Comment:
should we be more specific, i.e. `vscode-desktop-extension` in case if we switch to public API in VS Code Browser, web or remote extensions?